### ISSUE TITLE
Users need to know when they’ve finished the transaction

### ISSUE THREAD

**Issue by petegale** [alphagov/verify-local-patterns#237]
**Impact:** Users who are uncertain of the status of their application will be more likely to contact their local authority to enquire.

**Evidence:** Some users (especially those who had opted out of email notification) were unsure whether they had completed everything

**Comment by petegale** [comment]
![screen shot 2017-03-13 at 11 27 18](https://cloud.githubusercontent.com/assets/2970019/23852718/1b5db622-07e1-11e7-85f0-9f38869212e5.png)

**Comment by suryanaga** [comment]
Ok, going to keep this one as the canonical "people didn't realise they were done" issue for Parking Permits.

#327 and #385 show this is still an issue for Sunderland and Northumberland. I'm not going to close those issues just yet as they could do with making their styling more like [the pattern](https://www.gov.uk/service-manual/design/confirmation-pages). (We know that works well on other gov services.)

Also worth noting here that this is currently still an active issue for Concessionary Travel, as documented in #352.

**Comment by petegale** [comment]
I think it's worth considering that the cause of this issue may not be to do with styling of the page, but how the transaction flow deviates from what the users expect.

I doubt that users are failing to see the completion message. Rather they don't think everything has happened yet, so question whether this is accurate.

This was flagged in the feedback call with Northumberland. They felt that users were expecting some sort of summary / confirmation page before they were told the transaction was complete.

I'll probe for more on this in research if possible.

**Comment by EUzkuraityte** [comment]
UR evidence from Oxfordshire:
Impact: Users are left feeling unsure they have successfully completed the process.
Evidence: “No payment successful screen.”
“Why do I want a reference number?”
“Payment confirmation – other online service end with this.”
“Want a payment receipt – would rather get this automatically than have to sign up for it.”

**Comment by lizziebruce** [comment]
@EUzkuraityte 

This seems like 2 issues, 1. unsure if payment made and 2. wanting a receipt automatically. 2. would be technical functionality but for 1. suggest copy changes:

> Page title: Payment confirmation
> 
> Body text to begin: Your application is complete. 

For the reference number query, a copy edit could help. I'll revisit this page!

**Comment by EUzkuraityte** [comment]
@lizziebruce Your suggested copy looks good to me. We haven't seen this issue today, so it's up to your discretion of how you want to proceed with this - implement the changes and test or just close it.

**Comment by lizziebruce** [comment]
@EUzkuraityte I can also see from the screenshot on this issue ^^^ that the prototype changed between the UR issue note on 13 March and the one we tested yesterday. I'd put this into see if this still comes up in research unless @petegale thinks it can now be completely closed.

**Comment by petegale** [comment]
We didn't see evidence of this in round 3.

**Comment by EUzkuraityte** [comment]
reopening this issue, as new evidence submitted from Sunderland indicate that users are still confused.

![image](https://user-images.githubusercontent.com/20285115/27860660-6b597a24-6175-11e7-83dc-4505030cb55c.png)

**Comment by joanneSouthern** [comment]
This was apparent in user research last week at Northumberland also, users seemed to want a 'thankyou'

**Comment by lizziebruce** [comment]
This could be a result of Sunderland/Northumberland styling being different from that recommended by the prototype e.g. a block of colour behind the reference number, as we haven't had this issue continue to occur when testing with Argleton:

Argleton
![screen shot 2017-07-05 at 11 39 55](https://user-images.githubusercontent.com/27814324/27860951-b125bd0a-6176-11e7-92a7-9d367f737c9c.png)

Sunderland
![screen shot 2017-07-05 at 11 36 46](https://user-images.githubusercontent.com/27814324/27860934-a3fdf552-6176-11e7-9ffc-ce3b43d6247d.png)

Northumberland
![screen shot 2017-07-05 at 11 44 50](https://user-images.githubusercontent.com/27814324/27861127-62354160-6177-11e7-8348-e6c404ab7b46.png)

To fix this universally, we could add a sentence of copy saying "Your application is complete." before "Your 12 months permit..." Benefits: adds clarity. Disadvantages: more text on page (clutter, less readable) – but only slightly more.

**Comment by lizziebruce** [comment]
@EUzkuraityte just to check, do Northumberland residents always not receive physical permits? If so, we need to re-do the Application complete page for Northumberland.

**Comment by EUzkuraityte** [comment]
Hi @joanneSouthern any chance you could answer Lizzie's query 'do Northumberland residents always not receive physical permits?'

**Comment by joanneSouthern** [comment]
Northumberland always get a physical permit through the post

**Comment by EUzkuraityte** [comment]
@joanneSouthern Thank you for clarifying! In the screenshot above it says 'you won't receive a physical permit' - I presume this is an error in the prototype?

**Comment by joanneSouthern** [comment]
@EUzkuraityte yes the text is confusing, it says you wont get a permit and then further down says you have to display a permit in your car, its never updated from when GDS changed the original prototype, i did message Sanjay and ask him if we had to do something to update the content in our prototype, not heard anything back yet.  We are also concentrating on trying to get the tech infrastructure set up rather then keep up with all the prototype changes, unfortunately not enough IT resource to do both at the moment

**Comment by lizziebruce** [comment]
@joanneSouthern @EUzkuraityte 
The prototype has now been updated to reflect physical permits for Northumberland:
![screen shot 2017-07-05 at 15 11 52](https://user-images.githubusercontent.com/27814324/27868365-69d782b2-6194-11e7-89e7-c288f1353dd9.png)

**Comment by lizziebruce** [comment]
And for everyone (including Northumberland) have just added a sub-header to make it very clear the application is now complete....
![screen shot 2017-07-05 at 15 17 15](https://user-images.githubusercontent.com/27814324/27868714-7fb672d6-6195-11e7-9169-d396110481f8.png)

**Comment by ingasiler** [comment]
UR finding from 11 July
![screen shot 2017-07-18 at 15 49 57](https://user-images.githubusercontent.com/19390851/28323464-ce871f1a-6bd0-11e7-87cf-977670bd9ebd.png)

**Comment by lizziebruce** [comment]
I've changed the copy in preceding payment screens which should help users realise they have applied for and paid for their permit.

Specifically:
"Payment summary" (was "Paying for your permits")
<img width="731" alt="screen shot 2017-09-08 at 10 45 44" src="https://user-images.githubusercontent.com/27814324/30206121-fd266148-9482-11e7-9248-40c70e873961.png">

No change
<img width="719" alt="screen shot 2017-09-08 at 10 45 51" src="https://user-images.githubusercontent.com/27814324/30206127-ffa8b5ce-9482-11e7-9c2c-e9c71c5268a1.png">

No change
<img width="726" alt="screen shot 2017-09-08 at 10 46 13" src="https://user-images.githubusercontent.com/27814324/30206133-0467ead0-9483-11e7-8d71-61c43e081992.png">

"Making your payment" – was "Confirm your payment 
"Pay now" CTA – was "Confirm payment" CTA
<img width="696" alt="screen shot 2017-09-08 at 10 44 27" src="https://user-images.githubusercontent.com/27814324/30206140-08557bc6-9483-11e7-8b0b-b20ca50d1c94.png">

I think the only other thing we could do to help users understanding would be to move the confirmation message and renewal message screens to before payment so that the journey is payment details, confirm payment, application complete. The rhythm is a little lost by the messages screens coming inbetween. @sanjaypoyzer any thoughts?

I don't think there is any copy that could further emphasise the application complete screen is the end of the application. It follows the GDS pattern for an application complete page which has been researched already.

**Comment by suryanaga** [comment]
From what I can see, every version that has resulted in people not feeling like the application was complete doesn't follow the pattern for that page
